src/main/java/com/datasonnet/commands/Run.java

```diff
@@ -84,8 +82,8 @@ private String payload() throws IOException {
         }
     }
 
-    private Map<String, Document> combinedArguments() throws IOException {
-        return new HashMap<String, Document>() {{
+    private Map<String, Document<?>> combinedArguments() throws IOException {
```

Comment:
Oh fun, git's editor swallowed part of what I wrote instead of escaping it. That's... not a good choice. But anyways:

...always returns `Document<String>`...

Reply:
Yes, I tried that, but then line 57 complains about wrong parameter type, because `mapper.transform()` expects `Map<String, Document<?>>` parameter.

Reply:
Ah ha, I think that's something to address. Those shouldn't take the unknown type, which is mostly for inline shorthand, they should take a (parametrized) type T.